docs/modules/operation/pages/quick-start/performance-data.adoc

```diff
@@ -0,0 +1,41 @@
+
+= Step 4: Set Up Performance Data Collection
+
+This section describes how to set up performance data collection in {page-component-title}.
+
+{page-component-title} collects performance data using xref:reference:daemons/daemon-config-files/collectd.adoc[collectd], which is enabled by default.
+Collectd schedules data collection on network entities using management agents and protocol-specific collectors.
+Data collection for SNMP and OpenNMS-JVM is enabled by default, allowing {page-component-title} to monitor itself through JMX.
+
+For additional information, including collector configuration, see xref:operation:deep-dive/performance-data-collection/introduction.adoc[Performance Management] in the Deep Dive section.
+
+== Before you begin
+
+{page-component-title} data collection for SNMP works out of the box, provided your SNMP string is configured correctly.
+
+The default value for the community string is `public`.
+If your community string is different, you need to change it by following these steps:
+
+. In the web UI, click the *Gear* symbol at the top-right of the screen.
+. Choose menu:Provisioning[Configure SNMP Community Names by IP Address].
+. In the v1/v2c Specific Parameters section, enter the new community string in the *Read Community String* box.
+. Click *Save Config* to save your changes.
+
+Performance data collection on other protocols (HTTPS, JMX, and so on) requires additional configuration.
+You may also want to change how collectd works—when, how, and what data it collects.
+For more information, see xref:reference:performance-data-collection/introduction.adoc[Collectors].
+
+== Default performance data collection
+
+By default, {page-component-title} collects a large variety of information from associated nodes.
```

Comment:
Let's say "from monitored nodes that support SNMP" here.

Reply:
```suggestion
By default, {page-component-title} collects a large variety of information from monitored nodes that support SNMP.
```